mypy/api.py

```diff
@@ -0,0 +1,56 @@
+'''

```

Comment:
Style nit: Use triple double quotes and the summary on the first line (example: `"""Single-line summary. ...`) for docstrings.